scripts/obtain/dotnet-install.ps1

```diff
@@ -173,23 +173,23 @@ function GetHTTPResponse([Uri] $Uri)
 }
 
 
-function Get-Latest-Version-Info([string]$AzureFeed, [string]$AzureChannel, [string]$CLIArchitecture) {
+function Get-Latest-Version-Info([string]$AzureFeed, [string]$Channel, [string]$CLIArchitecture) {
```

Comment:
What I'm saying here is that the `CLIArchitecture` is not used in this function.